WhaleSpotting/Repositories/WhaleSightingRepo.cs

```diff
@@ -0,0 +1,35 @@
+using WhaleSpotting.Models.Database;
+using WhaleSpotting.Models.Request;
+using Microsoft.EntityFrameworkCore;
+
+namespace WhaleSpotting.Repositories;
+
+public interface IWhaleSightingRepo
+{
+    WhaleSighting GetById(int id);
+}
+
+public class WhaleSightingRepo : IWhaleSightingRepo
+{
+    private readonly WhaleSpottingDbContext context;
+    public WhaleSightingRepo(WhaleSpottingDbContext context)
+    {
+        this.context = context;
+    }
+
+    public WhaleSighting GetById(int id)
+    {
+        try
+        {
+            return context.WhaleSightings
+                .Where(u => u.Id == id)
+                .Include(u => u.User)
+                .Include(u => u.WhaleSpecies)
```

Comment:
can you change the `u` here to be `ws` for whale sighting as the `u` here was taken for User